include/exadg/functions_and_boundary_conditions/function_cached.h

```diff
@@ -38,7 +38,7 @@ class FunctionCached
 private:
   typedef std::tuple<unsigned int /*face*/, unsigned int /*q*/, unsigned int /*v*/> Id;
   typedef std::map<Id, types::global_dof_index>                                     MapVectorIndex;
-  typedef std::vector<std::vector<Tensor<rank, dim, Number>>> ArrayVectorTensor;
+  typedef std::vector<Tensor<rank, dim, Number>>                                    ArrayTensors;
```

Comment:
The new name (with plural s) is not consistent with the other names that use the singular form, so I would suggest `ArrayTensor` for now.